examples/compiled/bar_aggregate_count.vg.json

```diff
@@ -101,7 +101,9 @@
       "title": "precipitation (binned)",
       "labelFlush": true,
       "labelOverlap": true,
-      "tickCount": {"signal": "ceil(width/20)"},
+      "tickCount": {
+        "signal": "(domain('x')[1] - domain('x')[0]) / bin_maxbins_10_precipitation_bins.step + 1"
```

Comment:
Why not `(bin_maxbins_10_precipitation_bins.stop - bin_maxbins_10_precipitation_bins.stop) / bin_maxbins_10_precipitation_bins.step`?

Reply:
I think you mean stop-start. Sure we can do that. 

In any case, we need plus 1 for sure.

Reply:
Actually, that makes the code a bit more complicated because for prebinned, we might have the bin signal and only have the step.  

See axis/properties.ts

![image](https://user-images.githubusercontent.com/111269/50408389-1ff88d00-081c-11e9-8077-687d12d0d541.png)